restapi/countries.go

```diff
@@ -0,0 +1,39 @@
+package restapi
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/go-chi/render"
+	"github.com/uwblueprint/shoe-project/internal/database/models"
+	"github.com/uwblueprint/shoe-project/restapi/rest"
+)
+
+func (api api) ReturnAllCountries(w http.ResponseWriter, r *http.Request) render.Renderer {
+	var countries []models.Country
+	err := api.database.Find(&countries).Error
+	if err != nil {
+		return rest.ErrInternal(api.logger, err)
+	}
+	names := make([]string, len(countries))
+	for i, t := range countries {
+		names[i] = t.Name
+	}
+	return rest.JSONStatusOK(names)
+}
+
+func (api api) AddCountries(w http.ResponseWriter, r *http.Request) render.Renderer {
```

Comment:
Can you document a sample payload here so frontend can use it as a reference when they call this endpoint?

Reply:
like in the code as a comment?